apicast/src/configuration.lua

```diff
@@ -142,14 +142,17 @@ function _M.parse_service(service)
       authentication_method = proxy.authentication_method or backend_version,
       hosts = proxy.hosts or { 'localhost' }, -- TODO: verify localhost is good default
       api_backend = proxy.api_backend,
-      error_auth_failed = proxy.error_auth_failed,
-      error_auth_missing = proxy.error_auth_missing,
-      auth_failed_headers = proxy.error_headers_auth_failed,
-      auth_missing_headers = proxy.error_headers_auth_missing,
-      error_no_match = proxy.error_no_match,
-      no_match_headers = proxy.error_headers_no_match,
+      error_auth_failed = proxy.error_auth_failed or 'Authentication failed',
+      error_limits_exceeded = proxy.error_limits_exceeded or 'Limits exceeded',
+      error_auth_missing = proxy.error_auth_missing or 'Authentication parameters missing',
+      auth_failed_headers = proxy.error_headers_auth_failed or 'application/json; charset=utf-8',
```

Comment:
This looks wrong. Because the default value is not a json. So the content type should be text/plain.

Reply:
:man_facepalming: right.

Reply:
Fixed.